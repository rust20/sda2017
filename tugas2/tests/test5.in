2 1 5 100 5 5
Saber;50;100;pedang
Archer;50;100;panah
2;5;Saber,Archer
1;3;299;10;pedang;2
1;4;9999;1;panah;1000
2;1;299;10;panah;2
3;5;199;1;pedang;1
5;4;201;5;pedang;1
##DDM
D#.#S
.#.#D
.#.#.
...D.

