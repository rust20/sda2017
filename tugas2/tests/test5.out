2,5 Pahlawan yang ikut:Archer,Saber
3,5 BATTLE, kekuatan: 200, pahlawan: Archer
5,4 RUN, kekuatan maksimal sejumlah: 200
1,3 BATTLE, kekuatan: 300, pahlawan: Archer,Saber
1,4 RUN, kekuatan maksimal sejumlah: 150
2,1 RUN, kekuatan maksimal sejumlah: 150
Akhir petualangan Gudako
Level Gudako: 22
Level pahlawan:
Archer: 12
Saber: 11
